plugins/source/onfleet/resources/services/tasks.go

```diff
@@ -0,0 +1,30 @@
+package services
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/onfleet/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	"github.com/keplr-team/go-onfleet/onfleet"
+)
+
+func Tasks() *schema.Table {
+	return &schema.Table{
+		Name:        "onfleet_tasks",
+		Resolver:    fetchTasks,
+		Description: "https://docs.onfleet.com/reference/list-tasks",
+		Transform: transformers.TransformWithStruct(&onfleet.Task{},
+			transformers.WithPrimaryKeys("Id"),
+			transformers.WithTypeTransformer(client.OnfleetTypeTransformer),
```

Comment:
You can use the 'SharedTransformers' pattern here, moving WithType and WithResolver to a single function in `client/transformers.go` and just specifying primarykeys as the option